components/KNavigationBar.vue

```diff
@@ -25,7 +25,7 @@
               <a
                 v-else
                 :key="item.label"
-                rel="noopener"
```

Comment:
add `rel="noopener"` that the page from which you are redirected can't track the website from which you are